t/23_utils_git.t

```diff
@@ -0,0 +1,38 @@
+use strict;
+use warnings;
+use testapi;
+use Test::MockModule;
+use Test::Exception;
+use Test::More;
+
+use Utils::Git;
+
+subtest '[git_clone] Compose command' => sub {
+    my $mock_git = Test::MockModule->new('Utils::Git', no_auto => 1);
+    my @calls;
+    $mock_git->redefine(assert_script_run => sub { @calls = $_[0]; return 0; });
+    $mock_git->redefine(record_info => sub { return 0; });
```

Comment:
```suggestion
    $mock_git->redefine(record_info => sub {  note(join(' ', 'RECORD_INFO -->', @_)); });
```
It's cool to see them on the UT output. This is not functional to the UT itself but nice to have in the UT execution output

Reply:
Yep. Agree. Fixed.